### PR TITLE
Changes to generate fastboot-testing.js for setting additional sanbox options

### PR DESCRIPTION
**Pull request by dnalagatla**
Made changes for #57

### CODE REVIEW

**Comment by ryanto** [comment]
Awesome! Thanks for working on this @dnalagatla !

Let me know if you want me to take a look at anything

**Comment by dnalagatla** [comment]
> Awesome! Thanks for working on this @dnalagatla !
> 
> Let me know if you want me to take a look at anything

Can you just check if the flow makes sense? What I did here is to ensure the config `fastboot-testing.js` is generated in the config location of the App. In most cases, the directory will be `config` but if config path is defined in `package.json`, then that file will be generated in that location.

In index.js, it is checking if the file `fastboot-testing.js` exists in the config path of the APP and load options from there. Please let me know if looks good. If this flow looks good for you can you merge. the changes.

**Comment by ryanto** [comment]
This looks great!

File: index.js [hunk 1]
```diff
@@ -128,15 +130,29 @@ module.exports = {
...
       this.fastboot.reload({
         distPath
       });
     } else {
-      this.fastboot = new FastBoot({
+      let fastbootOptions = Object.assign({
         distPath
-      });
+      }, options);
+      debugger;
```
Comment (ryanto) [inline index.js]: Left over debugger :)

Comment (dnalagatla) [reply index.js]: that was miss. Fixed it. Thank you :)

**Comment by ryanto** [comment]
Thanks, this looks great @dnalagatla! 👍 

Looks like we had dependabot merge some PRs since this was opened, and there's now a conflict in yarn.lock. If you pull in master and `yarn install` it should auto resolve.

**Comment by dnalagatla** [comment]
> Thanks, this looks great @dnalagatla! 👍
> 
> Looks like we had dependabot merge some PRs since this was opened, and there's now a conflict in yarn.lock. If you pull in master and `yarn install` it should auto resolve.

Thanks @ryanto. I did a rebase with master.

File: blueprints/fastboot-test-config/files/__config__/fastboot-testing.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  resilient: false,
+  sandboxGlobals: {}
+};
```
Comment (CvX) [inline blueprints/fastboot-test-config/files/__config__/fastboot-testing.js]:
```suggestion
};

```

File: tests/dummy/config/fastboot-testing.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  resilient: false,
+  sandboxGlobals: {
+    SampleGlobal: `TestSampleGlobal`
+  }
+};
```
Comment (CvX) [inline tests/dummy/config/fastboot-testing.js]:
```suggestion
};

```

File: blueprints/fastboot-test/index.js [hunk 4]
```diff
@@ -1,9 +1,8 @@
 /* eslint-env node */
 module.exports = {
-  description: 'A test that runs against fastboot generated html',
+  description: 'Generates test config for adding custom Fastboot sanbox globals',
```
Comment (CvX) [inline blueprints/fastboot-test/index.js]: Descriptions for `blueprints/fastboot-test/index.js` and `blueprints/fastboot-test-config/index.js` seem swapped around?

Also, there's a typo: `sanbox`.

File: yarn.lock [hunk 5]
```diff
@@ -5,14 +5,12 @@
 "@babel/code-frame@^7.0.0":
   version "7.0.0"
   resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.0.0.tgz#06e2ab19bdb535385559aabb5ba59729482800f8"
-  integrity sha512-OfC2uemaknXr87bdLUkWog7nYuliM9Ij5HUcajsVcMCpQrcLmtxRbVFTIqmcSkSeYRBFBRxs2FiUqFJDLdiebA==
```
Comment (CvX) [inline yarn.lock]: All those accidental `yarn.lock` changes should be reverted.

File: index.js [hunk 6]
```diff
@@ -128,15 +130,28 @@ module.exports = {
...
+    let configPath = 'config';
+    let pkg = this.project.pkg;
+    if (pkg['ember-addon'] && pkg['ember-addon']['configPath']) {
+      configPath = pkg['ember-addon']['configPath'];
+    }
+
+    let fastbootTestConfigPath = path.resolve(configPath, 'fastboot-testing.js');
+
+    let options = {};
+    if(fs.existsSync(fastbootTestConfigPath)) {
```
Comment (CvX) [inline index.js]:
```suggestion
    if (fs.existsSync(fastbootTestConfigPath)) {
```

File: blueprints/fastboot-test-config/files/__config__/fastboot-testing.js (hunk 2 above)
Comment (dnalagatla) [reply blueprints/fastboot-test-config/files/__config__/fastboot-testing.js]: Thanks @CvX.  I fixed it.

File: index.js (hunk 6 above)
Comment (dnalagatla) [reply index.js]: Fixed it, thank you

File: blueprints/fastboot-test-config/files/__config__/fastboot-testing.js (hunk 2 above)
Comment (dnalagatla) [reply blueprints/fastboot-test-config/files/__config__/fastboot-testing.js]: fixed it.

**Comment by ryanto** [comment]
Alright this is published as `0.1.0`. Thanks again @dnalagatla

**Comment by dnalagatla** [comment]
Thank you @ryanto

**Pull request merged**